src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -995,13 +986,32 @@ public Locale getLocale() {
 
   @Override
   public Map<String, Locale> getLocales() {

```

Comment:
Seems like this should return a set, not a map..  no need to change now.. it just seems odd..  or if it's a map, i'd expect some way to lookup a which bots have which locale..

Reply:
you can - botInfo has a mrl Locale